src/python/bot/tasks/fuzz_task.py

```diff
@@ -1255,7 +1255,7 @@ def get_strategy_distribution_from_ndb():
   for strategy_entry in list(ndb_utils.get_all_from_query(query)):
     distribution.append({
         "strategy_name": strategy_entry.strategy_name,
-        "probability": strategy_entry.probability
+        "probability": strategy_entry.probability_medium_temperature
```

Comment:
We should probably have some kind of smarter fallback in the event that we don't find any data. There might be realistic use cases for this (new deployments, etc.) where we want to go back to the default distribution.